tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -0,0 +1,382 @@
+            3. Select the created ec2 CR.
+            4. Click Compute Profile tab.
+            5. Edit (3-Large) with valid configurations and submit.
+
+        :expectedresults: The compute resource created and opened successfully
+
+        :Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_retrieve_ec2_vm_list(self):
+        """List the virtual machines from ec2 compute resource.
+
+        :id: a8f09878-4aec-484c-a9d6-710ffe2cc8df
+
+        :setup: ec2 hostname and credentials.
+
+        :steps:
+
+            1. Select the created compute resource.
+            2. Go to "Virtual Machines" tab.
+
+        :expectedresults: The Virtual machines should be displayed
+
+        :Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_ec2_host_with_image(self):
+        """ Provision a host on ec2 compute resource with image based
+
+        :id: 5b68bfb8-6c15-4629-b25a-8928afdc7ca9
+
+        :setup: ec2 hostname and credentials.
+            1. Addition of custom provisioning template
+               to add ssh-key parameter.
+
+        :steps:
+            1. Populate images into satellite from ec2 images.
+            2. Associate Activation key and CV to the host.
+            3. Edit the required Operating system to use the custom
+               provisioning template.
+            4. Go to "Hosts --> New host".
+            5. Fill in the required details.(eg name,loc, org)(hostgroup)
+            6. Select ec2 compute resource from "Deploy on" drop down.
+            7. Associate appropriate feature capsules.
+            8. Go to "operating system tab".
+            9. Select the appropriate image .
+            10. Associate the activation key and submit.
+
+        :expectedresults: The host should be provisioned successfully
+
+        :Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_ec2_with_compute_profile(self):
+        """ Provision a host on ec2 compute resource with compute profile
+        default (3-Large)
+
+        :id: 01c8b5de-2b37-4cb2-989c-fae5a9446944
+
+        :setup: ec2 hostname, credentials and provisioning setup.
+
+        :steps:
+            1. Go to "Hosts --> New host".
+            2. Fill in the required details.(eg name,loc, org).
+            3. Select ec2 compute resource from "Deploy on" drop down.
+            4. Select the "Compute profile" from the drop down.
+            5. Provision the host using the compute profile.
+
+        :expectedresults: The host should be provisioned successfully
+
+        :Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_ec2_with_custom_compute_settings(self):
+        """ Provision a host on ec2 compute resource with
+         custom flavour and disk.
```

Comment:
I doubt, if we can update the disk to a custom value, can you please check this.